src/modules/vmount/vmount.cpp

```diff
@@ -220,7 +220,7 @@ static int vmount_thread_main(int argc, char *argv[])
 	ControlData *control_data = nullptr;
 	g_thread_data = &thread_data;
 
-	int last_active = 0;
+	int last_active = -1;
```

Comment:
From just reading through the code it doesn't seem right. Because further down we use:
```
bool already_active = (last_active == 1);
```
and clearly that's wrong from a wording perspective. So if we initialize with `-1` then this reads correctly.